src/engine/renderer/gl_shader.cpp

```diff
@@ -2173,6 +2174,27 @@ void GLShader::WriteUniformsToBuffer( uint32_t* buffer ) {
 	}
 }
 
+GLShader_generic::GLShader_generic( GLShaderManager *manager ) :
+	GLShader( "generic", ATTR_POSITION | ATTR_TEXCOORD | ATTR_QTANGENT, manager ),
+	u_ColorMap( this ),
+	u_DepthMap( this ),
+	u_TextureMatrix( this ),
+	u_ModelMatrix( this ),
+	u_ModelViewProjectionMatrix( this ),
+	u_ColorModulate( this ),
+	u_Color( this ),
+	u_ProfilerZero( this ),
+	u_ProfilerRenderSubGroups( this ),
```

Comment:
You don't really need these 2 uniforms for the non-3D shader. I guess you could technically use it with UI, but I don't see much benefit to it.

Reply:
Done.